### ISSUE TITLE
Can't get it running 

### ISSUE THREAD

**Issue by 20Bosko07** [tannercollin/sharenote-py#1]
Hello!
Unfortunately I can't get it to run in Docker. The Docker container is running, but when I go to http://host:8086 I get a blank page. If I then try to upload a note, it first does something (longer than if no connection) and then says error uploading. Can someone help me?

Best regards

**Comment by 20Bosko07** [comment]
Found this error:

File "/usr/local/lib/python3.11/site-packages/flask_cors/extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sharenote-py/main.py", line 175, in create_note
    title = data['template']['title']
            ~~~~~~~~~~~~~~~~^^^^^^^^^
KeyError: 'title'

**Comment by tanner37** [comment]
Can you confirm that encryption is disabled in the Share Note plugin settings?

**Comment by 20Bosko07** [comment]
@tannercollin thank you sooo much now it startet working....
but it looks a littel bit ugly:

![image](https://github.com/user-attachments/assets/138fb152-a34e-4ded-be5c-e506503a1fd9)

Any ideas?

**Comment by tanner37** [comment]
Can you open the Obsidian command palate (ctrl+p) and type "force" and force reupload of all data? Then check again.

If that doesn't work, can you show me if there's any errors in your browser's network inspector?

**Comment by 20Bosko07** [comment]
This didn't work. Here:
![image](https://github.com/user-attachments/assets/63b020a5-0c01-48c4-870b-31a112ed1bf1)

**Comment by 20Bosko07** [comment]
Ah, maybe I know the problem. Everywhere in the code is localhost, also in Obsidian it generates a localhost link. This could be the problem.

![image](https://github.com/user-attachments/assets/27fca3aa-ccdc-4afe-b44b-cf8d0fe206e9)
![image](https://github.com/user-attachments/assets/a0f9a663-2f40-4428-aef8-397d1d73a24c)

**Comment by tanner37** [comment]
I can't seem to reproduce. Can you show me what's in your `static/` folder?

It should work fine on localhost:

![image](https://github.com/user-attachments/assets/d13ca31a-8c29-45de-9a56-049aefbfe46f)

**Comment by 20Bosko07** [comment]
Yeah, but it's on a other Server, not on my pc. I think the problem is, he tries to download the files (picture, etc.) from localhost, and this is not possible. 

But here is the static folder:
![image](https://github.com/user-attachments/assets/2a1daabb-035a-4df7-9111-bb5950ca7ba3)

**Comment by tanner37** [comment]
You'll need to use the server's IP or a domain and set in your settings.py

**Comment by 20Bosko07** [comment]
Yes it's already set:

![image](https://github.com/user-attachments/assets/a99605cb-02bd-4607-b6df-e89576039a5a)

**Comment by tanner37** [comment]
Can you try rebuilding the docker image, and also doing a force reupload in Obsidian?

```
sudo docker compose down
sudo docker compose build sharenote
sudo docker compose up
```
